plugins/clipboard-manager/guest-js/index.ts

```diff
@@ -53,4 +54,60 @@ async function readText(): Promise<string> {
   return kind.plainText.text;
 }
 
-export { writeText, readText };
+/**
+ * Writes HTML or fallbacks to write provided plain text to the clipboard.
+ * @example
+ * ```typescript
+ * import { writeHtml, readHtml } from '@tauri-apps/plugin-clipboard-manager';
+ * await writeHtml('<h1>Tauri is awesome!</h1>', 'plaintext');
+ * await writeHtml('<h1>Tauri is awesome!</h1>', '<h1>Tauri is awesome</h1>'); // Will write "<h1>Tauri is awesome</h1>" as plain text
+ * assert(await readHtml(), '<h1>Tauri is awesome!</h1>');
+ * ```
+ *
+ * @returns A promise indicating the success or failure of the operation.
+ *
+ * @since 2.0.0
+ */
+async function writeHtml(
+    html: string,
+    alt_html: string
+): Promise<void> {
+  return invoke("plugin:clipboard-manager|write_html", {
+    data: {
+      html: {
+        html,
+        alt_html
+      },
+    },
+  });
+}
+
+/**
+ * Gets the clipboard content as HTML text.
+ * @example
+ * ```typescript
+ * import { readHtml } from '@tauri-apps/plugin-clipboard-manager';
+ * const clipboardHtml = await readHtml();
+ * ```
+ * @since 2.0.0
+ */
+async function readHtml(): Promise<object> {
```

Comment:
We need a better return type here, seems like it is returning a string

Reply:
In this case i opted to returning entire object as i wanted to provide entire result back, but yea i agree as there is no way in arboard to determine if set_html() fallback is used instead of actual html alt will always be null, since arboard also does not have get_html() as it has for plaintext and images.

Reply:
hmmm, we could instead remove this function and rely on `get_text`, I think it is enough

Reply:
Will remove it right now

Reply:
Pushed a change stripping everything related to read_html from both sides. Will mark this as resolved.